app/src/components/governance/ParticipantList.vue

```diff
@@ -94,7 +94,7 @@ h4 {
 .participant div {
   display: flex;
   align-items: center;
-  width: 2rem;
+  width: 5rem;
```

Comment:
what is this for?

Reply:
yeah, the current width was wrong, it creates a horizontal scroll and hides the "NoWithVeto" and "Abstain" votes

Reply:
you can just delete the width i think

Reply:
then you lose straight lines for columns

Reply:
no we'll solve it a different way